src/chapter1/linux.md

```diff
@@ -1 +1,56 @@
 # Linux Setup
+
+For your training you will need a few tools. Namely:
+
+- Git
+- C compiler toolchain (GCC)
+- A text editor (VSCode)
+
+ For this section I will be assuming a Debian system, namely Ubuntu. Replace `apt` commands with your distributions relevant package manager.
```

Comment:
When I have time, I will update this section with instructions for RHEL/Fedora family which is what I personally use, and also Arch because I've seen couple people using this too.